lib/takeaway.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'message'
+
+class TakeAway
+  attr_reader :menu, :basket, :order_total, :message
+
+  def initialize(message = Message.new)
+    @menu = {
+            "Sweet & Sour Chicken Balls" => 3.99,
+            "Chicken Chowmein" => 4.50,
+            "Crispy Shredded Beef" => 5.90,
+            "Egg Fried Rice" => 3.00,
+            "Fried Duck with Black Bean Sauce" => 6.40
+            }
+    @basket = {}
+    @order_total = 0
+    @message = message
+  end
+
+  def read_menu
```

Comment:
This class has a few responsibilities. It
- Holds the menu information
- Manages the order